### DISCUSSION TITLE
Armor and Damage parameter in Simulation Controls

### DISCUSSION THREAD

**Discussion by xkortex** [MaxRobinsonTheGreat/EvolutionSimulatorV2#37] [Ideas and Suggestions]
Hi, first of all, awesome game! Been having a blast playing around with it.

I noticed in a lot of games, plants would evolve into "diatoms" with armor and killers, and pretty much dominate, with low complexity 3-cell "flies" periodically blooming on the food from overcrowding. It's challenging to grow really high complexity animals because the plants basically can just form a plaque. 

I think it would be beneficial to be able to control the damage of killer cells and protectiveness of armor, e.g. default of `1.0` damage and `0.0` armor multiplier. This would allow tuning the benefits of either cells and favor offensive vs defensive, without resorting to "One hit kill". I think Simulation Controls would be a logical place to put it in. 

Basically I'd just modify the `killNeighbor` cell to do a damage calc, and `harm` to take the value of that damage, rather than just `damage++`. 

https://github.com/MaxRobinsonTheGreat/LifeEngine/blob/2ef721682a1086d6659a6865bf2bef8fef25c0d5/src/Organism/Cell/BodyCells/KillerCell.js#L20-L30

https://github.com/MaxRobinsonTheGreat/LifeEngine/blob/2ef721682a1086d6659a6865bf2bef8fef25c0d5/src/Organism/Organism.js#L260-L265

Personally, I'd stick the controls after food drop probability, but I think that's less critical: 

https://github.com/MaxRobinsonTheGreat/LifeEngine/blob/master/src/Controllers/ControlPanel.js#L176

**Comment by doo158** [comment]
really i'd take anything to get away from these little fucks

🔳
🟦 🟧
